### PR TITLE
add Wikidata entries

### LINKED ISSUE #1012: Research needed for: shop/bakery|Marie Blachère
#### :thinking: We need help deciding what to do with "shop/bakery|Marie Blachère"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/bakery|Marie Blachère` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/bakery|Marie Blachère`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/bakery|Marie Blachère`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1013: Research needed for: shop/bakery|Mlinar
#### :thinking: We need help deciding what to do with "shop/bakery|Mlinar"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/bakery|Mlinar` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/bakery|Mlinar`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/bakery|Mlinar`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1015: Research needed for: shop/bakery|Nothing Bundt Cakes
#### :thinking: We need help deciding what to do with "shop/bakery|Nothing Bundt Cakes"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/bakery|Nothing Bundt Cakes` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/bakery|Nothing Bundt Cakes`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/bakery|Nothing Bundt Cakes`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1580: Research needed for: shop/optician|Opticalia
#### :thinking: We need help deciding what to do with "shop/optician|Opticalia"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/optician|Opticalia` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/optician|Opticalia`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/optician|Opticalia`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### LINKED ISSUE #1605: Research needed for: shop/shoes|Chaussea
#### :thinking: We need help deciding what to do with "shop/shoes|Chaussea"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/shoes|Chaussea` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/shoes|Chaussea`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/shoes|Chaussea`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### PR DESCRIPTION
**Pull request by Adamant36**
Added some Wikidata entries I created, fixed another one and also added some official names. Along with adding a few operators. Closes #1605, closes #1580, closes #1012, closes #1013, closes #1015.

### CODE REVIEW

File: brands/amenity/fast_food.json [hunk 1]
```diff
@@ -211,7 +211,10 @@
       "brand:wikidata": "Q177054",
       "brand:wikipedia": "en:Burger King",
       "cuisine": "burger",
-      "name": "Burger King"
+      "name": "Burger King",
+      "operator": "Restaurant Brands International",
```
Comment (bhousel) [inline brands/amenity/fast_food.json]: wait, aren't most of these operated by franchisees?
`operator` isn't really for "parent company"

File: brands/amenity/restaurant.json [hunk 2]
```diff
@@ -1374,7 +1374,10 @@
       "brand:wikidata": "Q846301",
       "brand:wikipedia": "en:Red Lobster",
       "cuisine": "seafood",
-      "name": "Red Lobster"
+      "name": "Red Lobster",
+      "operator": "Golden Gate Capital",
```
Comment (bhousel) [inline brands/amenity/restaurant.json]: same here

File: brands/amenity/fast_food.json (hunk 1 above)
Comment (Adamant36) [reply brands/amenity/fast_food.json]: Probably. I'll remove them. I'm probably a little loser in my usage of the operator tag then I could be, but there's no owner:wikipedia tag and Wikipedia linking is the main reason I use it :(

**Comment by Adamant36** [comment]
It should be fixed now.

**Comment by bhousel** [comment]
Thanks!

File: brands/shop/bakery.json [hunk 3]
```diff
@@ -429,6 +431,7 @@
     "countryCodes": ["ca", "us"],
     "tags": {
       "brand": "Nothing Bundt Cakes",
+      "brand:wikidata": "Q62082526",
```
Comment (tas50) [inline brands/shop/bakery.json]: Thanks for this one. I had it on my todo list

**Pull request merged**